src/Engine/Task/Factory/AATTaskFactory.cpp

```diff
@@ -45,6 +45,7 @@ static constexpr LegalPointSet aat_im_types{
   TaskPointFactoryType::AAT_SEGMENT,
   TaskPointFactoryType::AAT_ANNULAR_SECTOR,
   TaskPointFactoryType::AAT_KEYHOLE,
+  TaskPointFactoryType::AAT_VARIABLE_KEYHOLE_SECTOR,
```

Comment:
```
src/Engine/Task/Factory/AATTaskFactory.cpp:48:25: error: no member named 'AAT_VARIABLE_KEYHOLE_SECTOR' in 'TaskPointFactoryType'
```

Reply:
Huh? Are you sure you had the working directory up to date with the HEAD?